source/find_getmore_killcursors_commands.rst

```diff
@@ -88,218 +88,6 @@ find
 
 The **find** command replaces the query functionality of the OP_QUERY wire protocol message but cannot execute queries against special collections. Unlike the legacy OP_QUERY wire protocol message, the **find** command cannot be used to execute other commands.
 
-.. code:: javascript
```

Comment:
Since this is being deleted, can we change the first instance of `**find**` in the paragraph above to link to https://docs.mongodb.com/manual/reference/command/find/?